suitcase/fields.py

```diff
@@ -8,6 +8,7 @@
 import six
 from suitcase.exceptions import SuitcaseChecksumException, SuitcaseProgrammingError, \
     SuitcaseParseError, SuitcaseException, SuitcasePackStructException
+import suitcase
```

Comment:
That could be resolved by adding `from suitcase.structure import Structure` to the setval method.